Application/src/Main.java

```diff
@@ -30,12 +30,18 @@ public void start(Stage primaryStage) {
             public void handle(ActionEvent event) {
                 Collection<String> result = Import.file(primaryStage);
                 if (result == null) return;
+
+                
+
 				// Check if the JSON file contains a supported diagram type.
                 System.out.println(DiagramCheck.ContainsDiagram(result));
-                // TODO remove print line and parse result (user story 5)
-                System.out.println(result);
+                // Parse the element if it contains a sequence_diagram
+				if (DiagramCheck.ContainsDiagram(result) == "sequence_diagram" ){
```

Comment:
String == String?
Do you mean String.equals(String)?
Because I don't think this even works as you think it would.

Reply:
you're right, i will change it. But in some cases it will work that way
https://stackoverflow.com/questions/9698260/what-makes-reference-comparison-work-for-some-strings-in-java